tools/install.sh

```diff
@@ -54,49 +53,60 @@ main() {
-      exit 1
+      err "Windows/MSYS Git is not supported on Cygwin"
+      die "Make sure the Cygwin git package is installed and is first on the path"
     fi
   fi
-  env git clone --depth=1 https://github.com/ohmybash/oh-my-bash.git $OSH || {
-    printf "Error: git clone of oh-my-bash repo failed\n"
-    exit 1
-  }
+  env git clone --depth=1 https://github.com/ohmybash/oh-my-bash.git "${OSH}" \
+    || die "git clone of oh-my-bash repo failed"
 
-  printf "${BLUE}Looking for an existing bash config...${NORMAL}\n"
-  if [ -f $HOME/.bashrc ] || [ -h $HOME/.bashrc ]; then
-    printf "${YELLOW}Found ~/.bashrc.${NORMAL} ${GREEN}Backing up to ~/.bashrc.pre-oh-my-bash${NORMAL}\n";
-    mv $HOME/.bashrc $HOME/.bashrc.pre-oh-my-bash;
+  msg "${BLUE}Looking for an existing bash config...${NORMAL}\n"
+  if [ -f "${HOME}/.bashrc" ] || [ -h "${HOME}/.bashrc" ]; then
+    msg "${YELLOW}Found ~/.bashrc.${NORMAL}"
+    msg "${GREEN}Backing up to ~/.bashrc.pre-oh-my-bash${NORMAL}"
+    mv "${HOME}/.bashrc" "${HOME}/.bashrc.pre-oh-my-bash"
   fi
 
-  printf "${BLUE}Using the Oh My Bash template file and adding it to ~/.bashrc${NORMAL}\n"
-  cp $OSH/templates/bashrc.osh-template $HOME/.bashrc
-  sed "/^export OSH=/ c\\
-export OSH=$OSH
-  " $HOME/.bashrc > $HOME/.bashrc-ombtemp
-  mv -f $HOME/.bashrc-ombtemp $HOME/.bashrc
+  msg "${BLUE}Using the Oh My Bash template file and adding it to ~/.bashrc${NORMAL}"
+  cp "${OSH}/templates/bashrc.osh-template" "${HOME}/.bashrc"
+
+  # update bashrc template to point to osh
+  sed -i "s/^export OSH=.*$/export OSH=${OSH}" "${HOME}/.bashrc"
 
   # MOTD message :)
   printf '%s' "$GREEN"
-  printf '%s\n' '         __                          __               __  '
-  printf '%s\n' '  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_ '
-  printf '%s\n' ' / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \'
-  printf '%s\n' '/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / /'
-  printf '%s\n' '\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/ '
-  printf '%s\n' '                        /____/                            .... is now installed!'
-  printf "%s\n" "Please look over the ~/.bashrc file to select plugins, themes, and options"
-  printf "${BLUE}${BOLD}%s${NORMAL}\n" "To keep up on the latest news and updates, follow us on GitHub: https://github.com/ohmybash/oh-my-bash"
-  exec bash; source $HOME/.bashrc
+  printf '%s\n' \
+  '         __                          __               __   ' \
+  '  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_  ' \
+  ' / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \ ' \
+  '/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / / ' \
+  '\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/  ' \
+  '                        /____/                            .... is now installed!' \
+  "Please look over the ~/.bashrc file to select plugins, themes, and options"
```

Comment:
I am now thinking of picking this change.

Reply:
Go right ahead. Personally I like the consistency of `msg`, `err`, `die`, but they aren't necessary.

Reply:
Thank you! I have rebased and merged it.